docs/guide/global.md

```diff
@@ -0,0 +1,59 @@
+# The global doc
```

Comment:
```suggestion
# Global settings
```

Reply:
I get that this is a doc, but talking about it as a doc in the headline is inside baseball a bit. I know that real programmers will get it, but there is a middle ground.